packages/react/src/components/RoomInformation/RoomInformation.js

```diff
@@ -1,5 +1,6 @@
 import React, { useContext } from 'react';
-import { Avatar } from '@rocket.chat/fuselage';
+//import { Avatar } from '@rocket.chat/fuselage';
```

Comment:
Hey @KushalSr You can safely remove this comment.

Reply:
Hi @abhinavkrin , actually I'm having some issues with my laptop. Can you wait till evening, I'll remove the comment or if it's urgent then I've to use someone's else laptop.
Thanks

Reply:
Hey @KushalSr no worries take your time.

Reply:
Thanks @abhinavkrin

Reply:
@abhinavkrin Hi Abhinav. I've remove the comment check once.
Thanks

Reply:
Hey @KushalSr ,  Have you checked if the image of the room is being rendered or not?

Reply:
The image of the room is not being rendered. Could you set size="100%" in line 55 , like `<Avatar size="100%" url={getChannelAvatarURL(channelInfo.name)} />`

Reply:
@abhinavkrin okay, I'll work on that